tools/perf-automation/Azure.Sdk.Tools.PerfAutomation/Program.cs

```diff
@@ -13,6 +13,7 @@
 using System.Threading.Tasks;
 using YamlDotNet.Core;
 using YamlDotNet.Serialization;
+using static Azure.Sdk.Tools.PerfAutomation.Program;
```

Comment:
Is this needed?

Reply:
Doesn't appear so, I need to properly set up my .NET dev environment to clean this up automatically.